src/shipping/shipping-request-options.ts

```diff
@@ -2,6 +2,13 @@ import { RequestOptions } from '../common/http-request';
 
 import { AmazonPayShippingInitializeOptions } from './strategies/amazon';
 
+/*
+ * A set of options for configuring requests related to the default shipping method.
+ */
+export interface UpdateShippingAddressRequestOptions extends RequestOptions {
+    includeShippingOptions?: boolean;
```

Comment:
I think the current way of passing optional includes is just an `include` flag, like `CheckoutParams`. And we pass the option like this.

```ts
loadCheckout('123', { params: { include: CheckoutIncludes.PhysicalItemsCategoryNames } })
```

Do you want to introduce a new convention?